### PR TITLE
CLIO-39530: First pass on thCheckbox a11y

### PR DESCRIPTION
**Pull request by craigcarlyle**

### CODE REVIEW

File: src/lib/thCheckbox/examples/5 - Accessibility/coffee.coffee [hunk 1]
```diff
@@ -0,0 +1,3 @@
+angular.module('thDemo', ['ThemisComponents'])
```
Comment (JeffMarvin) [inline src/lib/thCheckbox/examples/5 - Accessibility/coffee.coffee]: Double quotes please.

File: src/lib/thCheckbox/thCheckbox.directive.coffee [hunk 2]
```diff
@@ -1,26 +1,31 @@
+keycode = require "keycode"
```
Comment (JeffMarvin) [inline src/lib/thCheckbox/thCheckbox.directive.coffee]: Sweet! I didn't know about this, but I like it.

File: src/lib/thCheckbox/thCheckbox.directive.coffee [hunk 3]
```diff
@@ -1,26 +1,31 @@
...
         $scope.$apply =>
           @checked = not @checked
         @change() if @change?
 
-      $element.on 'click', =>
+      $element.on "click", =>
         @indeterminate = no
         @toggle() unless $element.attr("disabled")
 
+      $element.on "keyup", (event) ->
```
Comment (JeffMarvin) [inline src/lib/thCheckbox/thCheckbox.directive.coffee]: Curious, why do we use keyup in particular?

File: src/lib/thCheckbox/examples/5 - Accessibility/coffee.coffee (hunk 1 above)
Comment (craigcarlyle) [inline src/lib/thCheckbox/examples/5 - Accessibility/coffee.coffee]: Addressed.

File: src/lib/thCheckbox/thCheckbox.directive.coffee (hunk 2 above)
Comment (craigcarlyle) [inline src/lib/thCheckbox/thCheckbox.directive.coffee]: 👍

File: src/lib/thCheckbox/thCheckbox.directive.coffee (hunk 3 above)
Comment (craigcarlyle) [inline src/lib/thCheckbox/thCheckbox.directive.coffee]: It seems to play nicer with screen readers over keydown.

**Pull request merged**